curiouslearning/FeedTheMonsterJS
# Somali Language  Build

# Changes
- Somali json and audios

# How to test
- Check for the missing audios and feedback texts.

Ref: [CR-66](https://curiouslearning.atlassian.net/jira/software/projects/CR/boards/7?selectedIssue=CR-66)


[CR-66]: https://curiouslearning.atlassian.net/browse/CR-66?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
